interface/patient_tracker/patient_tracker.php

```diff
@@ -1098,12 +1098,15 @@ function kiosk_FLB() {
                 $("#kiosk").val('');
                 $("[name='kiosk_hide']").show();
                 $("[name='kiosk_show']").hide();
+                console.log('Window')
                 //alert(e.type);
             }, false);
             addEventListener('fullscreen', function (e) {
                 $("#kiosk").val('1');
                 $("[name='kiosk_hide']").hide();
                 $("[name='kiosk_show']").show();
+                console.log('全')
+                $('table').addClass('bg-dark')
```

Comment:
I don't think u meant to add this JS here

Reply:
Thanks for your mention, I removed the code.